src/basic/process-util.h

```diff
@@ -127,6 +127,8 @@ static inline bool sched_priority_is_valid(int i) {
         return i >= 0 && i <= sched_get_priority_max(SCHED_RR);
 }
 
+#define PID_INVALID 0
```

Comment:
maybe at least do `((pid_t) 0)` so that `typeof(PID_INVALID)` resolves to the same type as `pid_t`